manifests/params.pp

```diff
@@ -358,6 +438,7 @@
       $client_package_name      = 'snmp'
       $client_config            = '/etc/snmp/snmp.conf'
 
+      $trap_service_name        = 'snmptrapd'
```

Comment:
What does this have to do with strict vars?  I believe it is missing because tested versions of Debian do not have it.